multi-agent/lib/mas/core/field_hints.py

```
@@ -76,18 +82,18 @@ class ActionHint(BaseModel):
         description="Whether the action supports search filtering (has search_regex param)"
     )
-    on_success: Optional["ActionHint"] = Field(
+    on_success: "ActionHint | None" = Field(
         default=None,
         description="Action to fire after this action succeeds. "
                     "The chained action receives its own dependencies from the form. "
                     "Skipped when any required dependency value is empty.",
     )
 
-    def model_dump(self, **kwargs) -> Dict[str, Any]:
+    def model_dump(self, **kwargs: Any) -> dict[str, Any]:
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**kwargs`

(ANN401)